lib/liblastlog2.map

```diff
@@ -1,5 +1,6 @@
-LIBLASTLOG2_1.0 {
+LIBLASTLOG2_1.2 {
```

Comment:
That's not correct, as this would change all symbol versions.
Correct would be:
```
LIBLASTLOG2_1.0 {
  global:
        ll2_read_all;
        ll2_read_entry;
        ll2_remove_entry;
        ll2_rename_user;
        ll2_write_entry;
        ll2_update_login_time;
        ll2_import_lastlog;
  local: *;
};
LIBLASTLOG2_1.2 {
  global:
        ll2_check_database;
} LIBLASTLOG2_1.0;
```

Reply:
Indeed. I hope the last modification is suitable for the version script file.